example/reader_util.h

```diff
@@ -336,4 +336,15 @@ float ndpi_flow_get_byte_count_entropy(const uint32_t byte_count[256], unsigned
 
 extern int nDPI_LogLevel;
 
+/* TODO: conditional compilation? */
```

Comment:
Would've make sense to me. Either use `NDPI_ENABLE_DEBUG_MESSAGES`, `DEBUG_TRACE` or `DEBUG_PAYLOAD`. It's up to you. I would recommend the first one as it should be set with `./configure --enable-debug-messages`.

Reply:
Ok, thanks for the suggestion